src/core/index.ts

```diff
@@ -402,17 +402,26 @@ export class Core {
             expiration: U64 | number | string;
         } & (
             | {
-                  lockScriptHash: H160 | string;
-                  parameters: Buffer[];
+                  lockScriptHashFrom: H160 | string;
+                  parametersFrom: Buffer[];
               }
             | {
-                  recipient: AssetTransferAddress | string;
-              })
+                  recipientFrom: AssetTransferAddress | string;
+              }) &
+            (
+                | {
+                      lockScriptHashFee: H160 | string;
+                      parametersFee: Buffer[];
+                  }
+                | {
+                      recipientFee: AssetTransferAddress | string;
+                  }
+                | {})
     ): Order {
         const {
             assetTypeFrom,
             assetTypeTo,
-            assetTypeFee = "0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
FYI, `H256.zero()` is now available since primitives-js@0.4.4

Reply:
Nice! I'll fix this

Reply:
No, I'll put in this PR. I think it's too minor and does not matter